concepts/basics/about.md

```diff
@@ -65,7 +65,7 @@ So we'll start with this.
 module Calculator ( add ) where
 ```
 
-http://learnyouahaskell.com/modules#making-our-own-modules
+http://learnyouahaskell.github.io/modules#making-our-own-modules
```

Comment:
consider making all http links https instead. When I attempt to visit the link using http, I am redirected to https already, so we might as well save the redirect.

Reply:
I did this by accident but reverted it. Should I replace all `http`s with `https`, even in non-LYAH urls?